app/assets/stylesheets/nav_bar.css.scss

```diff
@@ -1,30 +1,33 @@
 @import url("//fonts.googleapis.com/css?family=Impact");
 
 .nav-padding {
-  padding-left: 3.5em;
-  padding-right: 3.5em;
+  padding: 0 3%;
 }
 
 .title {
+  background-color: #d9b82d;
+  border-radius: 0.1em;
   font-family: Impact;
   font-size: 2.4em;
-  background-color: #d9b82d;
   text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
-  border-radius: 0.1em;
 }
 
 .top-bar .name .title-small {
-  width: 4em;
+  background-color: #d9b82d;
```

Comment:
Color literals like `#d9b82d` should only be used in variable declarations; they should be referred to via variable everywhere else.